install/kubernetes/helm/istio/values.yaml

```diff
@@ -386,10 +386,10 @@ global:
   defaultResources:
     requests:
       cpu: 10m
-    #   memory: 128Mi
-    # limits:
-    #   cpu: 100m
-    #   memory: 128Mi
+      memory: 128Mi
+    limits:
+      cpu: 100m
+      memory: 128Mi
```

Comment:
this is super dangerous. If a limit isn't explicitly defined for pilot it will get this low limit and almost certainly get OOM killed

Reply:
I think we should have a limit explicitly defined. besides the reason describes here - #12344, the autoscale does not work even if we enable it by default.

Reply:
autoscale works based on requests, not limits afaik

Reply:
> Then, if a target utilization value is set, the controller calculates the utilization value as a percentage of the equivalent resource request on the containers in each pod

It seems the limit is not useful if enable HPA. right?

Reply:
Yes, please do not set such low limits. HPA , placement works on requests.

Reply:
We do set cpu/mem limits for pilot by default, so this global default won't be used.

We have resource config for each components and sidecar pretty much... do we still need the defaultResources?  it is basically not really used.

Reply:
I see no reason for the defaults, as long as we explicitly provide settings for every component. I think there are problems if we don't define requests since some people have policies requiring them, etc

Reply:
if we can ensure every component provides such settings, the we can remove the global defaultResources safely.